### PR TITLE
Improve lobby UI

### PR DESCRIPTION
**Pull request by Kwasow**

### CODE REVIEW

File: app/src/main/java/com/pokerio/app/utils/GameState.kt [hunk 1]
```diff
@@ -25,14 +25,15 @@ object GameState {
...
+    var isPlayerAdmin: Boolean = false
 
     // Callbacks
     private var playerJoinedCallbacks = HashMap<Int, (Player) -> Unit>()
     private var playerRemovedCallbacks = HashMap<Int, (Player) -> Unit>()
     private var nextId = 0
 
     // Constants
-    const val BASE_URL = "http://158.101.160.143:42069"
+    const val BASE_URL = "http://192.168.86.30:42069"
```
Comment (sormys) [inline app/src/main/java/com/pokerio/app/utils/GameState.kt]: Was the change of address intentional?

Comment (Kwasow) [reply app/src/main/java/com/pokerio/app/utils/GameState.kt]: My bad, restored the original address

File: app/src/main/java/com/pokerio/app/screens/LobbyScreen.kt [hunk 2]
```diff
@@ -2,55 +2,167 @@ package com.pokerio.app.screens
...
+        GameState.addOnPlayerJoinedCallback {
+            numberOfPlayers = GameState.players.size
+            isAdmin = GameState.isPlayerAdmin
+        }
     DisposableEffect(LocalLifecycleOwner.current) {
         onDispose {
             // Unregister callback when we leave the view
             GameState.removeOnPlayerJoinedCallback(callbackId)
+            // Notify the server we're leaving the game
+            leaveGame(context)
```
Comment (sormys) [inline app/src/main/java/com/pokerio/app/screens/LobbyScreen.kt:61]: I believe that when we will go into settings screen this will activate.

**Review by sormys — APPROVED** [review]

File: app/src/main/java/com/pokerio/app/screens/LobbyScreen.kt (hunk 2 above)
Comment (sormys) [inline app/src/main/java/com/pokerio/app/screens/LobbyScreen.kt]: I believe that when we will go into settings screen this will activate.

**Review by sormys — CHANGES_REQUESTED** [review]

**Pull request merged**